core/common/test/samples/LocalDateRangeSamples.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019-2024 JetBrains s.r.o. and contributors.
+ * Use of this source code is governed by the Apache 2.0 License that can be found in the LICENSE.txt file.
+ */
+
+package kotlinx.datetime.test.samples
+
+import kotlinx.datetime.*
+import kotlin.random.Random
+import kotlin.test.Test
+
+class LocalDateRangeSamples {
+    val Jan_01_2000 = LocalDate(2000, 1, 1)
+    val Jan_05_2000 = LocalDate(2000, 1, 5)
+    val Jan_06_2000 = LocalDate(2000, 1, 6)
```

Comment:
The samples are broken, as they will not include these: 
![image](https://github.com/user-attachments/assets/93b2d1fe-0aa3-4129-a72f-ba7fa564cc27)
How to check this: `./gradlew dokkaHtml`, then open `./core/build/dokka/html/kotlinx-datetime/kotlinx.datetime/-local-date/range-to.html`